.gitignore

```diff
@@ -150,3 +150,37 @@ captures/
 
 # Remove after this framework is published on NPM
 code-push-plugin-testing-framework/node_modules
+
```

Comment:
Can you mirror these changes to the `.npmignore` file, and if possible, could you also leave a comment at the top of this file informing future contributors that they need to do the same?

I'm also assuming there's nothing you want to exclude from the npm package that isn't already listed here, but please add them there if so!